src/view/screens/Search/Search.tsx

```diff
@@ -965,8 +958,11 @@ function SearchHistory({
                       style={styles.profileAvatar as StyleProp<ImageStyle>}
                       accessibilityIgnoresInvertColors
                     />
-                    <Text style={[pal.text, styles.profileName]}>
-                      {truncateText(profile.displayName || '', 12)}
+                    <Text
+                      style={[pal.text, styles.profileName]}
+                      numberOfLines={1}
+                      ellipsizeMode="head">
```

Comment:
Is there a reason for setting this to `head`? My understanding is that (although your screenshot doesn't show this happening!) `head` will result in something like `helloworld123` being truncated like `...world123`. I think we'd rather the front of the string always be visible, so you'd see `hellowo...`.

Reply:
Cool! Yea it's a bit weird to me as well sometimes. I think the differences in android and ios text rendering are fairly different so you see a lot of things that just do not line up...